english/twilight-zone/blackjack.md

```diff
@@ -0,0 +1,9 @@
+You sit at the blackjack table. The dealer gives you an ace. Then a king. You have a blackjack! You win! You feel like you're in heaven!
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.